python/servo/command_base.py

```diff
@@ -457,11 +459,21 @@ def get_nightly_binary_path(self, nightly_date):
             print("The nightly file {} has already been extracted.".format(
                 destination_folder))
         else:
-            print("Extracting to {}...".format(destination_folder))
-            with tarfile.open(os.path.join(nightlies_folder, destination_file), "r") as tar:
-                tar.extractall(destination_folder)
-
-        return path.join(destination_folder, "servo", "servo")
+            print("Extracting to {} ...".format(destination_folder))
+            if is_windows():
+                command = 'msiexec /a {} /qn TARGETDIR={}' \
+                    .format(os.path.join(nightlies_folder, destination_file), destination_folder)
+                if subprocess.call(command, stdout=PIPE, stderr=PIPE) != 0:
+                    print("Could not extract the nightly executable from the msi package.")
+                    sys.exit(1)
+            else:
+                with tarfile.open(os.path.join(nightlies_folder, destination_file), "r") as tar:
+                    tar.extractall(destination_folder)
+        bin_folder = path.join(destination_folder, "servo")
+        if is_windows():
+            bin_folder = path.join(destination_folder, "PFiles", "Mozilla research", "Servo Tech Demo")
+        print(path.join(bin_folder, "servo{}".format(BIN_SUFFIX)))
```

Comment:
Do we still need this?

Reply:
I think I could extract the .msi archive to %temp% and just keep the .exe file instead :)

Reply:
I was asking about the `print`, but it's worth a try!

Reply:
Oh I see, no it's debug code I use (I seem to get the right path but when I try to ./mach run with a path to the executable it just won't ^^
I guess as soon as I figure it out I'll remove the WIP tag on the PR (Probably this weekend :))